doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -29,9 +29,8 @@ The following Kconfig options are also available for this module:
 
 * :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_ACCOUNT_KEY_MAX` - The option configures maximum number of stored Account Keys.
 * :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_TINYCRYPT`, :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_MBEDTLS`, and :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_OBERON` - These options are used to select the cryptographic backend for Fast Pair.
-  MbedTLS is used by default, whereas Tinycrypt is used by default for cases of building with TF-M as the Secure Execution Environment (:kconfig:option:`CONFIG_BUILD_WITH_TFM`).
-  This is because in such case the MbedTLS API cannot be directly used by the Fast Pair service.
-  The Oberon backend can be used to limit memory consumption.
+  The Oberon backend is used by default.
+  The Mbed TLS backend is using Mbed TLS crypto APIs which is considered a legacy API.
```

Comment:
```suggestion
  The Mbed TLS backend uses the Mbed TLS crypto APIs, which are now considered legacy APIs.
```

Progressive tense to present (is using -> uses), subject verb agreement (APIs vs API, is -> are).